### PR TITLE
Phase2-sim120 Add test to check hits in partial wafers for V17 geometry

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add test to check hits in partial wafers for V17 geometry

#### PR validation:

Use the test script in SimG4CMS/Calo/test/python to do the checking

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test with #38547

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimG4CMS/Calo/python/hgcalHitPartial_cff.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from SimG4CMS.Calo.hgcalHitPartialEE_cfi import *
+
+hgcalHitPartialHE = hgcalHitPartialEE.clone(
+    nameSense  = cms.string("HGCalHESiliconSensitive"),
+    caloHitSource = cms.string("HGCHitsHEfront"))
```
Comment (perrotta) [inline SimG4CMS/Calo/python/hgcalHitPartial_cff.py:7]:
```suggestion
    nameSense  = "HGCalHESiliconSensitive",
    caloHitSource = "HGCHitsHEfront"
)
```

**Review by perrotta — COMMENTED** [review]
`cmsRun runHGC7_cfg.py` ended up without crashes
Just fix the config in a safer way, as suggested here, and it will be merged

**Comment by bsunanda** [comment]
@perrotta Made the changes as suggested

**Comment by civanch** [comment]
+1

**Pull request merged**